backend/migrations/0001_initial.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.3 on 2020-11-07 19:15
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Rule',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('rule_name', models.CharField(blank=True, max_length=50, null=True)),
```

Comment:
```suggestion
                ('name', models.CharField(blank=True, max_length=50, null=True)),
```
Not nullable or blank. It's the Rule model, so no need to prefix every column with "rule_". Length sounds good I think, just make sure we know.

Reply:
Same applies for 3 below. Amount is also a number. See the story for details on the columns we need.

Reply:
Updated the names to remove rule_